src/script.js

```diff
@@ -477,7 +477,7 @@ window.onload = function() {
 
     chrome.storage.sync.get("loggedUser", function(userDetails) {
         var log = document.getElementById("log");
-        if (accessToken && userDetails.loggedUser.email) {
```

Comment:
or should it be `accessToken || userDetails.loggedUser.email`? But as per my understanding, `accessToken` can't be set before Line number 482. So we can totally remove this check.

Reply:
Yeah, thanks for the PR. :100: